headers/functions/overlay29.h

```diff
@@ -34,6 +34,7 @@ bool ShouldDisplayEntityWrapper(struct entity* entity);
 bool CanSeeTarget(struct entity* user, struct entity* target);
 bool CanTargetEntity(struct entity* user, struct entity* target);
 bool CanTargetPosition(struct entity* monster, struct position* position);
+void PopulateActiveMonsterPtrs();
```

Comment:
If there's no params, do

```suggestion
void PopulateActiveMonsterPtrs(void);
```

Putting () is only correct in C++; in C it technically means something else.